### PR TITLE
Add suffixes to typos from "af" to "al"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -2038,19 +2121,34 @@ agrandized->aggrandized
 agrandizes->aggrandizes
 agrandizing->aggrandizing
 agravate->aggravate
+agravated->aggravated
+agravates->aggravates
+agravating->aggravating
 agre->agree
+agreable->agreeable
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2128]:
```
agreably->agreeably
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -2038,19 +2121,34 @@ agrandized->aggrandized
...
 agred->agreed
+agreee->agree
+agreeeable->agreeable
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2131]:
```
agreeeably->agreeably
```

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -1977,46 +1981,125 @@ afterw->after
...
 aggreate->aggregate
 aggreated->aggregated
+aggreates->aggregates
+aggreating->aggregating
 aggreation->aggregation
 aggreations->aggregations
+aggreator->aggregator
+aggreators->aggregators
+aggree->agree
+aggreeable->agreeable
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2057]:
```
aggreeably->agreeably
```

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -2725,7 +2925,9 @@ altogehter->altogether
 altough->although
 altought->although
 altready->already
+altso->also
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2928]: Perhaps:
```
altso->also, alto,
```

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -2725,7 +2925,9 @@ altogehter->altogether
...
 alue->value
+alues->values
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2930]: Could it be a misspelling of one of the following?
```
aloes, blues, clues, flues, glues,
```

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:2128]: Done. I also added `agreeableness`.

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt]: I added `altso->altos`. I couldn't find `altso->alto`.

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt]: I couldn't find any of these.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**